src/main/kotlin/io/github/mojira/arisa/modules/AttachmentModule.kt

```diff
@@ -19,33 +19,21 @@ class AttachmentModule(
             val functions = attachments
                 .filter { endsWithBlacklistedExtensionAdapter(it.name) }
             assertNotEmpty(functions).bind()
-            val usernames = functions.getUsernames()
-            val attachmentNames = functions.getAttachmentNames()
+            val commentInfo = functions.getCommentInfo()
             functions
                 .map { it.remove }
                 .forEach { it.invoke() }
             addComment(CommentOptions(attachmentRemovedMessage))
-            addRawRestrictedComment("Attachment Details:\nFilename: $attachmentNames\nUploader: $usernames", "helper")
+            addRawRestrictedComment("Attachments:\n$commentInfo", "helper")
```

Comment:
Might be good to make it a little bit clearer what this comment describes:
```suggestion
            addRawRestrictedComment("Removed attachments:\n$commentInfo", "helper")
```